install/kubernetes/helm/istio/charts/gateways/templates/deployment.yaml

```diff
@@ -272,6 +272,7 @@ spec:
       {{- end }}
       affinity:
       {{- include "gatewaynodeaffinity" (dict "root" $ "nodeSelector" $spec.nodeSelector) | indent 6 }}
+      {{- include "podAntiAffinity" . | indent 6 }}
```

Comment:
There should be a scope issue here, you can' t generate `podAntiAffinity` correctly from current scope(`.`).

Reply:
i see, it is the other issue you mentioned https://github.com/istio/istio/pull/10123#issuecomment-472679152. Thank you, I'll rework.

Reply:
Hello! 

Any updates on this feature?

Thank you!